packages/web-client/src/webdav/client/builders.ts

```diff
@@ -41,6 +41,11 @@ export const buildPublicLinkAuthHeader = (password: string) => {
 
 export const buildAuthHeader = (token: string, space: SpaceResource = null): Headers => {
   if (isPublicSpaceResource(space)) {
+    // TODO: make check cleaner
+    if (space.driveAlias.startsWith('ocm/')) {
+      return { Authorization: `Bearer ${space.id}` }
```

Comment:
eh... what?! space id is the bearer token?!

Reply:
Yes ... ... ... ... ... ... ... .. .. ..

Reply:
the ocm token *must* be provided as bearer token (although it's laready part of the request url ...)

Reply:
That was the detail that I was still missing... ok, thanks for explaining. This is so horrible...